container/container_unix.go

```diff
@@ -415,6 +416,19 @@ func (container *Container) TmpfsMounts() []Mount {
 			Data:        data,
 		})
 	}
+	for _, mnt := range container.HostConfig.Mounts {
+		if mnt.Type == mounttypes.TypeTmpfs {
+			data := ""
+			if mnt.TmpfsOptions != nil {
+				data = mnt.TmpfsOptions.RawOptions
+			}
+			mounts = append(mounts, Mount{
+				Source:      "tmpfs",
```

Comment:
It is from the original source https://github.com/docker/docker/pull/26837/files#diff-5dfdabc4cd5100a2aee949c12d2afe6cL413